netlink-packet-utils/src/macros.rs

```diff
@@ -1,4 +1,4 @@
-#[macro_export]
+#[macro_export(local_inner_macros)]
```

Comment:
What does this do? I can't find the documentation for this attribute.

Reply:
There is documentation on https://doc.rust-lang.org/reference/macros-by-example.html

It makes all call to other macro from the same macro prefixed with $crate. This means you don't need to import all the macros and thus don't need #[macro_use] and extern crate